share/completions/bzr.fish

```diff
@@ -1,604 +1,156 @@
-complete -c bzr -l usage --description 'Show usage message and options'
-complete -c bzr -l verbose -s v --description 'Display more information'
-complete -c bzr -l help -s h --description 'Show help message'
-complete -c bzr -l quiet -s q --description 'Only display errors and warnings'
-complete -c bzr -l short --description 'Print just the version number'
-complete -c bzr -l usage --description 'Show usage message and options'
-complete -c bzr -l verbose -s v --description 'Display more information'
-complete -c bzr -l all --description 'Include all possible information'
-complete -c bzr -l check-clean --description 'Check if tree is clean'
-complete -c bzr -l format --description 'ARG Select the output format'
-complete -c bzr -l custom --description 'Version info in Custom template-based format'
-complete -c bzr -l python --description 'Version info in Python format'
-complete -c bzr -l rio --description 'Version info in RIO (simple text) format (default)'
-complete -c bzr -l help -s h --description 'Show help message'
-complete -c bzr -l include-file-revisions --description 'Include the last revision for each file'
-complete -c bzr -l include-history --description 'Include the revision-history'
-complete -c bzr -l quiet -s q --description 'Only display errors and warnings'
-complete -c bzr -l revision --description 'ARG, -r See "help revisionspec" for details'
-complete -c bzr -l template --description 'ARG Template for the output'
-complete -c bzr -l usage --description 'Show usage message and options'
-complete -c bzr -l verbose -s v --description 'Display more information'
-complete -c bzr -l template --description 'VERSION_INFO \"Project 1'
-complete -c bzr -l all --description 'Apply list or delete action to all views'
-complete -c bzr -l delete --description 'Delete the view'
-complete -c bzr -l help -s h --description 'Show help message'
-complete -c bzr -l name --description 'ARG Name of the view to define, list or delete'
-complete -c bzr -l quiet -s q --description 'Only display errors and warnings'
-complete -c bzr -l switch --description 'ARG Name of the view to switch to'
-complete -c bzr -l usage --description 'Show usage message and options'
-complete -c bzr -l verbose -s v --description 'Display more information'
-complete -c bzr -l branch --description 'Set identity for the current branch instead of globally'
-complete -c bzr -l directory --description 'ARG, -d Branch to operate on, instead of working directory'
-complete -c bzr -l email --description 'Display email address only'
-complete -c bzr -l help -s h --description 'Show help message'
-complete -c bzr -l quiet -s q --description 'Only display errors and warnings'
-complete -c bzr -l usage --description 'Show usage message and options'
-complete -c bzr -l verbose -s v --description 'Display more information'
+function __fish_bzr_needs_command
+    set cmd (commandline -opc)
+
+    if [ (count $cmd) -eq 1 -a $cmd[1] = 'bzr' ]
```

Comment:
What about this, introducing an intermediate step where you get the index in the list with the command name, and from there on just process as usual?